cockatrice/src/tab_deck_editor.cpp

```diff
@@ -863,6 +873,43 @@ void TabDeckEditor::actOpenCustomsetsFolder() {
 
 }
 
+void TabDeckEditor::actAddCustomSet()
+{
+
+#if QT_VERSION < 0x050000
+    QString dataDir = QDesktopServices::storageLocation(QDesktopServices::DataLocation);
+#else
+    QString dataDir = QStandardPaths::standardLocations(QStandardPaths::DataLocation).first();
```

Comment:
You don't need to fix it for this PR, but we should make a standard convenience utility that just returns the QString so we don't have to inline the `#if` checks all over.